### DISCUSSION TITLE
how to sound off when iOS is silent mode

### DISCUSSION THREAD

**Discussion by Devkjk** [airsdk/Adobe-Runtime-Support#3222] [Q&A]
I use Animate and release iOS app.
I dont care whether the iPhone is silentMode or not.
User says the sound is on when his iPhone is SilentMode.
I  need to care this now.
How to check silentmode?
Need any ANEs?

**Comment by ajwfrost** [comment]
Audio output from AIR should be controllable by the normal phone settings ... can I check what APIs / mechanisms you're using for playing the audio? Are you able to reproduce this yourself?

thanks

**Comment by Devkjk** [comment]
I use actionscript3 sound class only.
The user's phone is iPhone15, and I dont have it so ,I cant reproduce this.
My app was released about 2,3 years ago.
I think my app build may be too old .
